client/router.js

```diff
@@ -36,12 +37,20 @@ Router.prototype.route = function(path, options) {
   };
 
   this._routes.push(route);
+  if (options.name) {
+    this._pathsMap[options.name] = path;
```

Comment:
Can't we store the route by the name. Then we have the path object there.
What do you think?